src/app/core/auth.service.ts

```diff
@@ -22,10 +22,21 @@ export enum AUTH_GROUP {
   ADMIN
 }
 
+export interface Tokens {
```

Comment:
Yes, this is the login response. Wasn't set on naming, but went with `Tokens` because it matched the resource being created on login and registration and being requested during refresh.